nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml

```diff
@@ -24,6 +24,7 @@
         <ClearEntireCostmap service_name="local_costmap/clear_entirely_local_costmap"/>
         <ClearEntireCostmap service_name="global_costmap/clear_entirely_global_costmap"/>
         <Spin/>
+        <Wait wait_duration="5"/>
```

Comment:
You think the wait is more helpful as the last recovery rather than at the begining or after clearing the costmaps?

Reply:
In this particular context, yes. Since in this behavior tree we don't do a recovery then check if we're in a good place to continue working. We do all these sequentially so it really don't make sense to wait, then clear, spin and immediately try to plan. It would be better to do a bunch of "stuff" then wait for everything to update, then try again. 

I'd agree in the case where we try each then attempt to replan between waiting should be the very first

Reply:
OK